src/api/SampleProjectService/index.ts

```diff
@@ -0,0 +1,25 @@
+import { GetSampleProjectResponseDto } from "@/types/dto/SampleProjectService";
+
+/**
+ * SampleProjectService의 유즈케이스에서 사용할 fetch 함수를 정의합니다.
+ * 유즈케이스를 실제로 구현한 hook이 너무 커지는 것을 방지하기 위해 분리해두었습니다.
+ */
+
+/**
+ *
+ */
+export const getSampleProject = (
+  id: number,
+): Promise<GetSampleProjectResponseDto> =>
+  // 올바른 fetch 방식인지 테스트 필요합니다. 원래 상대경로로 적었으나 next.js 18 버전 이상은 절대경로만 인식한다는 얘기가 있어서 일단 이렇게 적었습니다.
+  fetch(`/api/mock/project/${id}`).then((res) =>
+    res.ok ? res.json() : Promise.reject(res),
+  );
```

Comment:
```suggestion
export const getSampleProject = (
  id: number,
) = (): Promise<GetSampleProjectResponseDto> =>
  // 올바른 fetch 방식인지 테스트 필요합니다. 원래 상대경로로 적었으나 next.js 18 버전 이상은 절대경로만 인식한다는 얘기가 있어서 일단 이렇게 적었습니다.
  fetch(`/api/mock/project/${id}`).then((res) =>
    res.ok ? res.json() : Promise.reject(res),
  );
```

queryFn으로 사용하려면 반드시 인자가 비어있는 함수여야하므로, id를 인자로 받아 `()=>fetch(~~)` 꼴의 함수를 리턴하는 `(id:number)=>()=>fetch(~~)`꼴의 고차 함수로 바꿔주어야 합니다.  #19 이슈에 이와 관련된 내용을 자세히 적어놓았으니 읽어보세요 :)

`postSampleProject`는 굳이 고차 함수일 필요 없습니다. useMutation의 mutationFn은 직접 콜하는 함수이므로, 인자를 가질 수 있기 때문입니다.